pkgconfig/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# CMakeLists.txt for libyui-mga/pkgconfig
+#
+# Support for pkg-config:
+#
+# Generate a libyui.pc file from libyui.pc.in and install it to
```

Comment:
NP: Copy&paste leftover (should be libyui-gtk)

Reply:
changed, thanks